### PR TITLE
PUB-1455 - Updated links

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1455

### Change description ###

Removed unneeded links. 
Added new ones to make a few bits dynamic
Removed unneeded templates

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/application.yaml [hunk 1]
```diff
@@ -70,6 +70,7 @@ notify:
     gov-guidance-page-link: ${NOTIFY_LINK_GOV_GUIDANCE:https://www.gov.uk}
     aad-pw-reset-link: ${NOTIFY_LINK_AAD_RESET_PW_LINK:https://pib2csbox.b2clogin.com/pib2csbox.onmicrosoft.com/oauth2/v2.0/authorize?p=B2C_1_initial_user_set_pwd&client_id=c7e6e2c6-c23c-48e8-b9f4-6bad25a95331&nonce=defaultNonce&redirect_uri=https%3A%2F%2Fpip-frontend.staging.platform.hmcts.net%2Flogin%2Freturn&scope=openid&response_type=id_token&prompt=login}
     media-verification-page-link: ${MEDIA_VERIFICATION_PAGE_LINK:https://pip-frontend.staging.platform.hmcts.net/media-verification?p=B2C_1_SignInMediaVerification}
+    admin-dashboard-link: ${ADMIN_DASHBOARD_LINK:https://pip-frontend.staging.platform.hmcts.net/admin-dashboard}
```
Comment (joshblackmoor) [inline src/main/resources/application.yaml:73]: Thinking in here we should update the defaults to the correct ones for staging and remove any that are no longer used

File: src/test/resources/application-test.yaml [hunk 2]
```diff
@@ -2,11 +2,13 @@ notify:
...
-    aad-sign-in-page-link: https://pib2csbox.b2clogin.com/pib2csbox.onmicrosoft.com/oauth2/v2.0/authorize?p=B2C_1_SignInUserFlow&client_id=c7e6e2c6-c23c-48e8-b9f4-6bad25a95331&nonce=defau&redirect_uri=https%3A%2F%2Fpip-frontend.staging.platform.hmcts.net%2Flogin%2Freturn&scope=openid&response_type=id_token&prompt=login
+    subscription-page-link: ${NOTIFY_LINK_SUBSCRIPTION_PAGE:https://pip-frontend.staging.platform.hmcts.net/otp-login-testing}
+    start-page-link: ${NOTIFY_LINK_START_PAGE:https://pip-frontend.staging.platform.hmcts.net}
+    aad-sign-in-page-link: ${NOTIFY_LINK_AAD_SIGNIN_LINK:https://pib2csbox.b2clogin.com/pib2csbox.onmicrosoft.com/oauth2/v2.0/authorize?p=B2C_1_SignInUserFlow&client_id=c7e6e2c6-c23c-48e8-b9f4-6bad25a95331&nonce=defau&redirect_uri=https%3A%2F%2Fpip-frontend.staging.platform.hmcts.net%2Flogin%2Freturn&scope=openid&response_type=id_token&prompt=login}
+    gov-guidance-page-link: ${NOTIFY_LINK_GOV_GUIDANCE:https://www.gov.uk}
```
Comment (joshblackmoor) [inline src/test/resources/application-test.yaml:8]: Same as the comment in application.yaml, may want to update the defaults in here to match

**Review by joshblackmoor — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml (hunk 1 above)
Comment (ChrisS1512) [reply src/main/resources/application.yaml:73]: Updated :) I have checked and all of these are used

File: src/test/resources/application-test.yaml (hunk 2 above)
Comment (ChrisS1512) [reply src/test/resources/application-test.yaml:8]: Updated :)

**Review by joshblackmoor — APPROVED** [review]

**Pull request merged**